pack/dwl/tmm.json

```diff
@@ -138,7 +138,7 @@
         "quantity": 2,
         "skill_willpower": 1,
         "slot": "Arcane",
-        "text": "Uses (5 charges).\n[action]<b> </b>Spend 1 charge: <b>Fight. </b>This attack uses [willpower] instead of [combat]. You get +1 [willpower] for this attack. If a [skull] symbol is revealed during this attack, this attack deals +2 damage.",
+        "text": "Uses (5 charges).\n[action] Spend 1 charge: <b>Fight. </b>This attack uses [willpower] instead of [combat]. You get +1 [willpower] for this attack. If a [skull] symbol is revealed during this attack, this attack deals +2 damage.",
```

Comment:
Since we're doing tidying - please can you move the closing bold tag up to right after 'Fight.' with a space afterwards?